python-libraries/nanover-core/tests/trajectory/test_frame_data.py

```diff
@@ -737,6 +737,17 @@ def test_copy_doesnt_mutate_original_array(value):
     assert frame.arrays["sample.new"][0] != "baby yoda"
 
 
+@pytest.mark.parametrize("typed_array", [[0], [0.0], ["string"]])
```

Comment:
Could `[]` be a valid value to test here? Or will protobuf not let you do that?

Reply:
That's a `ValueError` from out `FrameData` class because empty array can't determine type. I've raised this issue for possibilities of supporting it in the future: https://github.com/IRL2/nanover-protocol/issues/72